_docs/tasks/istio-auth.md

```diff
@@ -9,89 +9,100 @@ type: markdown
 Because each Istio CA is scoped within a namespace, Istio CAs in different namespaces will not interfere with each other
-and they are easy to clean up through a single command.
+and they are easy to [uninstall]({{home}}/docs/tasks/installing-istio.html#uninstalling).
 
-We have the YAML files *istio-auth-X.yaml* for deploying all Istio components including Istio CA into the namespace.
-Follow [the Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
+## Before you begin
 
-### Option 2: (recommended) using per-cluster CA
+This task assumes you have:
 
-Only a single Istio CA is deployed for the Kubernetes cluster, in a dedicated namespace.
-Doing this offers the following benefits:
+* Disabled Istio Auth (if you enabled it) for all namespaces in the Kubernetes cluster.
+Otherwise, this process will deploy a CA that conflicts with existing CAs.
 
-* In the near future, the dedicated namespace will use
-[Kubernetes RBAC](https://kubernetes.io/docs/admin/authorization/rbac/) (beta in Kubernetes V1.6) to provide security
-boundary. This will offer strong security for Istio CA.
+* Completed steps 1 - 3 in [the Istio installation guide]({{home}}/docs/tasks/installing-istio.html#installing-on-an-existing-cluster).
 
-* Services in the same Kubernetes cluster but different namespaces are able to talk to each other through Istio Auth
-without extra trust setup.
+## Enabling Istio Auth
 
-#### Deplying CA
+### Deploying CA
 
-The following command creates  namespace *istio-system* and deploys CA into the namespace:
+The Istio CA only needs to be deployed once for the cluster.
+The following command creates namespace *istio-system* and deploys CA into the namespace:
 
 ```bash
-kubectl apply -f ./kubernetes/istio-auth/istio-cluster-ca.yaml
+kubectl apply -f templates/istio-auth/istio-cluster-ca.yaml
 ```
 
-#### <a name="istioconfig"></a>Enabling Istio Auth in Istio config
+### Deploying other services
 
-The following command uncomments the line *authPolicy: MUTUAL_TLS* in the file *kubernetes/istio-X.yaml*,
-and backs up the original file as *istio-X.yaml.bak*
-(*X* corresponds to the Kubernetes server version, choose "15" or "16").
+The following command will enable mTLS for the services in the "default" namespace,
+and the services are able to use the per-cluster CA deployed in the last step.
+Use the parameter *-n yournamespace* to specify a namespace other than the default one.
 
 ```bash
-sed "s/# authPolicy: MUTUAL_TLS/authPolicy: MUTUAL_TLS/" ./kubernetes/istio-X.yaml > ./kubernetes/istio-auth-X.yaml
+kubectl apply -f templates/istio-auth/isio-auth-per-cluster-ca.yaml
 ```
 
-#### Deploying other services
-
-Follow [the general Istio installation guide](./installing-istio.html),
-and **choose "If you would like to enable Istio Auth" in step 3**.
+Follow [the general Istio installation guide]({{home}}/docs/tasks/installing-istio.html) from step 5.
```

Comment:
Instead of step number, add better the link to the anchor.
I notice we keep adding steps :(